app/api/tools_config.json

```diff
@@ -6,5 +6,13 @@
     "1": {
         "path": "features.dynamo.core",
         "metadata_file": "metadata.json"
+    },
+    "2": {
+        "path": "features.code_check.core",
```

Comment:
This feature is not required as it is not a part of the **Worksheet Generator** feature.